h2/src/test/org/h2/test/auth/TestAuthentication.java

```diff
@@ -39,35 +39,50 @@
  */
 public class TestAuthentication extends TestBase {
 
+    private static final String TESTXML = "<h2Auth allowUserRegistration=\"true\" createMissingRoles=\"false\">"
+            + "<realm name=\"ciao\" validatorClass=\"myclass\"/>"
+            + "<realm name=\"miao\" validatorClass=\"myclass1\">"
+            + "<property name=\"prop1\" value=\"value1\"/>"
+            + "<userToRolesMapper className=\"class1\">"
+            + "<property name=\"prop2\" value=\"value2\"/>"
+            + "</userToRolesMapper>"
+            + "</realm>"
+            + "</h2Auth>";
+
+    private String externalUserPassword;
+    private DefaultAuthenticator defaultAuthenticator;
+    private Session session;
+    private Database database;
+
+    /**
+     * Run just this test.
+     *
+     * @param a ignored
+     */
     public static void main(String... a) throws Exception {
         TestBase.createCaller().init().test();
     }
 
-    String externalUserPassword;
-
-
-    String getExternalUserPassword() {
+    private String getExternalUserPassword() {
```

Comment:
Synthetic access.

Reply:
Do you see the serious problem in synthetic access method at the tests?
I agree with with avoid synthetic access methods in production code because performance may be loosed (lets don't discuss JIT & inline).

Reply:
Yes, because they cause warnings in IDE if such check is enabled.

Reply:
Ok. Fixed.